### PR TITLE
Standup Meeting Updates

### PR DESCRIPTION
**Pull request by mhmaidi789**
- Upgraded sass
- Set default homepage of browser to google.com
- Fixed bug that resulted in mute button crashing app. 
- Removed redundant home button
- Refactored ternary statements

### CODE REVIEW

File: src/context/reducers/globalReducer.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { actionTypes } from '../actions/globalActions';
 export const GlobalContext = createContext(null);
 
 export const globalState = {
-  url: null,
+  url: "http://www.google.com/",
```
Comment (MaxWeisen) [inline src/context/reducers/globalReducer.js:7]: This is so cool! Simple and effective.

**Review by JIB3377 — APPROVED** [review]
Awesome. Our mvp is around the corner.

**Review by MaxWeisen — APPROVED** [review]
Great changes! I think we are on track to present our MVP this week.

**Review by dieunity — APPROVED** [review]
Great work! Really make the app less buggy

**Pull request merged**